virl/files/virl-openrc.sh

```diff
@@ -1,6 +1,6 @@
 export OS_USERNAME=admin
 export OS_PASSWORD=ADMIN_PASS
 export OS_TENANT_NAME=admin
-export OS_AUTH_URL=http://controller:35357/v2.0
+export OS_AUTH_URL=http://controller:35357/v3
```

Comment:
I don't see this file being used anywhere.  BTW you seem to be setting OS_SERVICE_ENDPOINT to v3 which breaks at least CLI keystone endpoint-list.